src/blas/backends/cublas/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(SOURCES cublas_level1.cpp
                 cublas_level3.cpp 
                 cublas_batch.cpp 
                 cublas_extensions.cpp 
+                $<$<BOOL:${BUILD_SHARED_LIBS}>: mkl_blas_cublas_wrappers.cpp>
```

Comment:
This change seems out of place for this rocBLAS PR. Is this an intentional change?

Reply:
I agree, but I will have to check if removing this will not break anything.

Reply:
what is the status here?

Reply:
Sorry, that file doesn't even exist anymore. removed from this PR